pages/mesosphere/dcos/1.14/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -248,6 +250,13 @@ Use this to set up a custom CA certificate. See [using a Custom CA Certificate](
 
 Use this to set up a custom CA certificate. See [using a Custom CA Certificate](/mesosphere/dcos/1.14/security/ent/tls-ssl/ca-custom#configuration-parameter-reference) page for a detailed configuration parameter reference. 
 
+### exhibitor_tls_required [enterprise type="inline" size="small" /]
+When required, failures during Exhibitor TLS bootstrap will prevent DC/OS from starting. By default, failures are logged and Exhibitor will fallback to insecure mode.
```

Comment:
Thanks. I just noticed that we should update our "hardening guide" so that it recommends setting `exhibitor_tls_required` to `true`.

https://docs.d2iq.com/mesosphere/dcos/1.13/security/ent/hardening/